docs/decorators/pagination/README.md

```diff
@@ -0,0 +1,70 @@
+Pagination Decorators
+=====================
+
+## Overview
+
+**Decorator Type:** `https://github.com/hyperledger/aries-toolbox/tree/master/docs/decorators/pagination`
+
+**Decorator:** `~paginate`, `~page`
+
+The pagination decorators defines elements common to messages querying for and
+returning a list of results where paging is desirable.
+
+## Decorator Definition
+
+### `~paginate`
+
+The `~paginate` decorator is used in messages querying for a list of results. If
+more than one element in the response message can be paginated, `~paginate` can
+be used as a suffix to indicate to which element it refers, i.e.
+`results~paginate`. This usage is outlined by message type definitions as
+required.
+
+Example:
+
+```jsonc
+{
+    "...",
+    "~paginate": {
+        "@type": ".../decorators/pagination/paginate",
```

Comment:
I'm not sure about the inclusion of `@type` in either decorator.

Reply:
I don't think we need to include the type here, Containing it within a decorator is enough.